content/20.pharmaceuticals.md

```diff
@@ -712,6 +712,24 @@ Participants reported mild and moderate systemic adverse events after the day 1
+During the Phase III trial, 43,538 participants were enrolled 1:1 in the placebo and the vaccine candidate and received two 30-μg doses 21 days apart [@doi:10.1056/NEJMoa2034577].
+Of these enrolled participants, 21,720 received BNT162b2 and 21,728 received a placebo [@doi:10.1056/NEJMoa2034577].
+Recruitment occurred at 135 sites across six countries: Argentina, Brazil, Germany, South Africa, Turkey, and the United States.
+An initial press release described the first 94 cases, which were consistent with 90% efficacy of the vaccine at 7 days following the second dose [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against].
+The release of the full trial information included additional cases and analyzed the first 170 cases occurring at least 7 days after the second dose, 8 of which occurred in patients who had received BNT162b2.
+The first interim analysis reported the results of the first 94 cases of COVID-19 occurring at least 7 days after the second dose.
+Of these, 8 occurred in participants who received BNT162b2 and 162 occurred
+The press release characterized the study population as diverse, reporting that 42% of the participants worldwide came from non-white backgrounds, including 10% Black and 26% Hispanic or Latino [@url:https://www.pfizer.com/science/coronavirus/vaccine].
+Within the United States, 10% and 13% of participants, respectively, identified themselves as having Black or Hispanic/Latino backgrounds [@url:https://www.pfizer.com/science/coronavirus/vaccine].
+Additionally, 41% of participants worldwide were 56 years of age or older [@url:https://www.pfizer.com/science/coronavirus/vaccine], and they reported that the efficacy of the vaccine in adults over 65 was 94% [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-conclude-phase-3-study-covid-19-vaccine].
+The primary efficacy analysis of the Phase III study was concluded on November 18, 2020 [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-conclude-phase-3-study-covid-19-vaccine], and the final results indicted 94% effectiveness of the vaccine.
+The vaccine was approved in early December by the United Kingdom's Medicines and Healthcare Products Regulatory Agency with administration outside of a clinical trial beginning on December 8, 2020 [@doi:10.1136/bmj.m4714; @url:https://www.bbc.com/news/uk-55227325].
+As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19].
+ 
 In summary, mRNA vaccines are promising tools in the prevention and control of pandemics because they may sidestep many challenges associated with traditional vaccine design and manufacturing.
```

Comment:
Just curious if you know after reading about these vaccines, why it is that these mRNA vaccines are more effective than say the flu vaccine? My understanding is that they are both inducing an immune response to prime your body, but the flu is using an attenuated/dead version of the vaccine while these mRNA vaccines are having your cells produce just the antigen (single protein, not even the full viral particle). So the end result is the same but the mechanism of delivery is different - at least that is my take

Reply:
That is also my understanding about the mechanism! I'm really not sure why they work so well, especially since I believe people were hoping for much lower efficacy (like 70%) based on just the mechanism, prior to testing! I can look into this but also tagging @RLordan in case he has any ideas!

Reply:
Sorry don't mean to give you more work. This was more a question that I was curious about in case you knew off hand. But not to say you should look into this, because I can certainly do that on my own:)

Reply:
I'll keep an eye out and let you know if I see anything!

Reply:
Sorry @rando2 @ajlee21 , I am late to the convo. You are correct with the mechanisms. In both cases you are showing the immune system what the target virus is using different methods. However, I think the greater efficacy comes from how the mRNA vaccine tends to lead to a greater response because the spike protein is made by the cells causing higher doses than a traditional adenovirus vector vaccine could achieve. That’s my take on it, I don’t think they were expecting such high efficacy, which means there will likely be further study into how long these vaccines can maintain their effects etc.